dipy/sims/tests/test_phantom.py

```diff
@@ -55,7 +55,9 @@ def test_phantom():
     FA[np.isnan(FA)] = 0
     # 686 -> expected FA given diffusivities of [1500, 400, 400]
     l1, l2, l3 = 1500e-6, 400e-6, 400e-6
-    expected_fa =  (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2-l3)**2 + (l3-l1)**2 )/np.sqrt(l1**2 + l2**2 + l3**2))
+    expected_fa = (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + 
+                   (l2 - l3)**2 + (l3 - l1)**2) / 
+                   np.sqrt(l1**2 + l2**2 + l3**2)))
```

Comment:
Yup. That's probably my bad for suggesting this. See failing tests: 

https://travis-ci.org/nipy/dipy/jobs/113234014#L2306